controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -106,7 +105,7 @@ func newReconciler(mgr manager.Manager, ci hcoutil.ClusterInfo, upgradeableCond
 	return &ReconcileHyperConverged{
 		client:               mgr.GetClient(),
 		scheme:               mgr.GetScheme(),
-		operandHandler:       operands.NewOperandHandler(mgr.GetClient(), mgr.GetScheme(), ci.IsOpenshift(), hcoutil.GetEventEmitter()),
+		operandHandler:       operands.NewOperandHandler(mgr.GetClient(), mgr.GetScheme(), ci.IsOpenshift(), ci.IsConsolePluginImageProvided(), hcoutil.GetEventEmitter()),
```

Comment:
What do you think about sending the whole `ci` object to the function? now there are 5 parameters already, some of them with a long name.

Reply:
Done.